packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [20.0.2]
+
+### Changed
+
+- Use new `@metamask/keyring-internal-api@^1.0.0`( [#4695](https://github.com/MetaMask/core/pull/4695))
+  - This package has been split out from the Keyring API.
+- Bump `@metamask/keyring-api` from `^10.1.0` to `^12.0.0` ([#4695](https://github.com/MetaMask/core/pull/4695))
+- Bump `@metamask/eth-snap-keyring` from `^5.0.1` to `^7.0.0` ([#4948](https://github.com/MetaMask/core/pull/4948))
```

Comment:
It looks like this PR also bumped the peer dependency on `@metamask/snaps-controller`: https://github.com/MetaMask/core/pull/4948/files#diff-11700fa807e8d925b649057d40f964047063e26926ce62fc5f09210de947dbffR65. This would count as a breaking change, so we would want to log that here and bump this package by a major version.

Out of curiosity, why did we bump the peer dependency? Did `snaps-controllers` 9.10.0 introduce new types that we now need to be compatible with?

Reply:
Actually, the PR you linked wrongly upgraded the `peerDependencies` yes, but we then reverted this back to `9.7.0` before releasing the `accounts-controller` (IDK if you recall this, but here's [the PR for the revert](https://github.com/MetaMask/core/pull/4959))

However, there's still a mistake here 😅 the PR used for this entry is incorrect 🙈 It should be:

```suggestion
- Bump `@metamask/eth-snap-keyring` from `^5.0.1` to `^7.0.0` ([#4695](https://github.com/MetaMask/core/pull/4695))
```

Reply:
Ha, right, I remember reviewing that now. It's been a long month 😅